hack/dockerfile/install/containerd.installer

```diff
@@ -4,7 +4,7 @@
 # containerd is also pinned in vendor.conf. When updating the binary
 # version you may also need to update the vendor version to pick up bug
 # fixes or new APIs.
-CONTAINERD_COMMIT=bb71b10fd8f58240ca47fbb579b9d1028eea7c84 # v1.2.5
+CONTAINERD_COMMIT=3a80a801af4254f5e6a351a75d7a0be9699b90ad
```

Comment:
Any specific reason to bump up containerd?

Reply:
I just follow the comment. If I understand it wrong, please let me know, thank you. 
https://github.com/moby/moby/blob/3f08ab936f0894842daf088b2ebdec0d0bb83cb3/vendor.conf#L79-L83

Reply:
This commit is coming from the master branch (so effectively is containerd 1.3.0-dev).

I'd prefer waiting for containerd 1.2.6 to arrive, and then bump both containerd and runc (otherwise the static docker binaries would have a different version than the `.rpm` / `.deb` packages that use `containerd.io` as a dependency)

Reply:
Thanks for your reply.  :+1: 
OK, I can hold this until containerd 1.2.6 release and then bump both them.

Reply:
:+1: perhaps you can;

- change the commit of the containerd binary ("install") to the tip of the containerd `release/1.2` branch
- use separate commits for "updating the containerd binary", "runc binary", and for vendor updates. Reason for that is that we backport the binary updates to release branches for docker, but not always have to bump the vendored code. Having separate commits allows us to cherry-pick them separately

Note that the vendored code of containerd currently uses a commit from master, but the containerd.installer should use the `release/1.2` branch


We also might want to wait for runc 1.0.0-rc8, as there was a regression in rc7; https://github.com/opencontainers/runc/pull/2031

Reply:
Thank you for your patient answer. :+1: 
I resubmitted once, I want you to help me confirm, is my practice correct?
Thanks again. :heart:

Reply:
I have open another PR #39016  as @thaJeztah said, "waiting for containerd 1.2.6 to arrive".